### PR TITLE
adding templates plus css for side menu

### PR DESCRIPTION
**Pull request by DmitriiCr**
image

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1 +1,5 @@
+# IDEs and editors
+/.idea
 
+# virtual environment
+/venv
```
Comment (FloPuz) [inline .gitignore]: '*.pyc' à mettre dans le gitignore 👯

File: .gitignore
Comment (VladimirChantitch) [inline .gitignore:1]: You should modify the standard python project gitignore

File: main.py
Comment (VladimirChantitch) [inline main.py:1]: Add python doc

File: project/__init__.py
Comment (VladimirChantitch) [inline project/__init__.py:1]: Move the from to the top of the file _and_remove the place holders

File: .gitignore (hunk 1 above)
Comment (FloPuz) [reply .gitignore]: Et aussi *.sqlite à rajouter dans le gitignore

File: project/db.sqlite
Comment (VladimirChantitch) [inline project/db.sqlite:1]: to add to git ignore

**Pull request merged**